registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -671,6 +674,7 @@
     "sha256:a780eabc911316b348f06fb02c43773b311a239b72496c24b47281ba1af456ba": ["v1.10.5"]
     "sha256:ad0ab0f68f638cd33635d63675b2068ea94799a7c3d51b61c524bf90bd7f9816": ["v1.10.6"]
     "sha256:43e3f156b48ebcf5317662b2773d7724655db78809f222b362783a5922da6d69": ["v1.10.7"]
+    "sha256:2954e2ae5e787aecb3a13f425e51f7c2e20deca93a20b6c36d78e0e47f31d8ce": ["v1.10.8"]
```

Comment:
SHA `2954e2ae5e78` matches kubeadm-control-plane-controller v1.10.8 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-control-plane-controller@sha256:2954e2ae5e787aecb3a13f425e51f7c2e20deca93a20b6c36d78e0e47f31d8ce/details).